RecoVertex/PrimaryVertexProducer/plugins/PrimaryVertexProducer.cc

```diff
@@ -451,7 +457,7 @@ void PrimaryVertexProducer::fillDescriptions(edm::ConfigurationDescriptions& des
   desc.add<bool>("useMVACut", false);
   desc.add<double>("minTrackTimeQuality", 0.8);
 
-  descriptions.add("primaryVertexProducer", desc);
+  descriptions.addDefault(desc);
```

Comment:
Same here - ConfDb parsing needs an explicit cfi file. I assume you want descriptions.addWithDefaultLabel ?!

Reply:
Fixed by #44288